### DISCUSSION TITLE
Why are the borders on input-bordered so faint now?

### DISCUSSION THREAD

**Discussion by rpungello** [saadeghi/daisyui#2047] [Q&A]
In daisyUI 2.x the borders applied via input-bordered were nice and visible, but with 3.x they're almost imperceptible to the point of not being useful.
Is there a reason for this change? Is it possible to customize things so the borders are a more appropriate color/opacity without having to create an entirely new class and refactor all code relying on input-bordered already?

**Comment by akkie** [comment]
https://github.com/saadeghi/daisyui/issues/2055 please vote
